index.html

```diff
@@ -34,236 +36,178 @@
-              </ul>
-              <button type='button' class="project-action">See project</button>
-            </div>
-            <div class="image-placeholder"></div>
-            
-           </li>
-           <li class="card">
-            <div class="card-content">
-              <h4 class="project-title text-primary text-center">Multi-Post Stories Gain+Glory</h4>
-              <ul class=" tag-language-container list-style-none ">
-                <li class="tag-language">Ruby on Rail</li>
-                <li class="tag-language">CSS</li>
-                <li class="tag-language">HTML</li>
-                <li class="tag-language">JavaScript</li>
-              </ul>
-              <button type='button' class="project-action">See project</button>
-            </div>
-            <div class="image-placeholder"></div>
-            
-           </li>
-           <li class="card">
-            <div class="card-content">
-              <h4 class="project-title text-primary text-center">Multi-Post Stories Gain+Glory</h4>
-              <ul class=" tag-language-container list-style-none ">
-                <li class="tag-language">Ruby on Rail</li>
-                <li class="tag-language">CSS</li>
-                <li class="tag-language">HTML</li>
-                <li class="tag-language">JavaScript</li>
-              </ul>
-              <button type='button' class="project-action">See project</button>
-            </div>
-            <div class="image-placeholder"></div>
-            
-           </li>
-           <li class="card">
-            <div class="card-content">
-              <h4 class="project-title text-primary text-center">Multi-Post Stories Gain+Glory</h4>
-              <ul class=" tag-language-container list-style-none ">
-                <li class="tag-language">Ruby on Rail</li>
-                <li class="tag-language">CSS</li>
-                <li class="tag-language">HTML</li>
-                <li class="tag-language">JavaScript</li>
-              </ul>
-              <button type='button' class="project-action">See project</button>
-            </div>
-            <div class="image-placeholder"></div>
-            
-           </li>
-           <li class="card">
-            <div class="card-content">
-              <h4 class="project-title text-primary text-center">Multi-Post Stories Gain+Glory</h4>
-              <ul class=" tag-language-container list-style-none ">
-                <li class="tag-language">Ruby on Rail</li>
-                <li class="tag-language">CSS</li>
-                <li class="tag-language">HTML</li>
-                <li class="tag-language">JavaScript</li>
-              </ul>
-              <button type='button' class="project-action">See project</button>
-            </div>
-            <div class="image-placeholder"></div>
-            
-           </li>
-          
-           
-        </ul>
-      </section>
-      <section id="about-me">
```

Comment:
The about section should also be converted to incorporate dynamic javascript to meet the project requirements for this task..
<img width="949" alt="Screen Shot 2023-05-25 at 9 16 59 AM" src="https://github.com/AsumVictor/Porfolio/assets/37291194/5afaab24-75be-465f-a4d0-1fd8486c0658">